pkg/registry/rbac/clusterrole/policybased/storage.go

```diff
@@ -46,6 +46,11 @@ func NewStorage(s rest.StandardStorage, authorizer authorizer.Authorizer, ruleRe
 	return &Storage{s, authorizer, ruleResolver}
 }
 
+// Destroy cleans up resources on shutdown.
+func (r *Storage) Destroy() {
+	// FIXME: Do we have to anything to destroy here?
```

Comment:
yes, the underlying ClusterRole storage (same for all the other rbac storage objects)

Reply:
done